### PR TITLE
Head lice review

### PR DESCRIPTION
**Pull request by Demotive**
Updating content after a cut-n-paste review with Kim.

head-lice-cutup

nits

### CODE REVIEW

**Comment by bottlerocket** [comment]
There is a colour contrast issue on the light-blue lede with the darker blue background. Do we need to fix that now?

**Comment by Demotive** [comment]
Yes we should. Make it a separate PR though, as it's going to affect various places.

File: views/conditions/head-lice.html [hunk 1]
```diff
@@ -25,102 +29,93 @@
...
-      <figure>
-          <img src="/images/head-lice/03-empty-eggs.jpg" alt="nits" />
-          <figcaption>The empty shells of the eggs after they hatch stay in the hair. These are called nits.</figcaption>
-      </figure>
-
+    <figure>
+      <img src="/images/head-lice/01_head_lice.png" alt="lice" />
+      <figcaption>Head lice are small insects, up to 3mm long.</figcaption>
+    </figure><!--
+    --><figure>
```
Comment (bottlerocket) [inline views/conditions/head-lice.html:36]: Is there a reaason why there are there comments in here?

Comment (Demotive) [inline views/conditions/head-lice.html:36]: Yes, it's one way to eradicate built-in element spacing when you're displaying `inline-block`. Have a quick look at https://css-tricks.com/fighting-the-space-between-inline-block-elements/

**Pull request merged**